catalog/transformer/catalog-transformer-xml/src/main/java/ddf/catalog/transformer/xml/adapter/AdaptedMetacard.java

```diff
@@ -71,7 +73,7 @@
   // Suppressing Warnings and using ArrayList rather than List here because
   // ArrayList implements Serializable (List does not).
   @SuppressWarnings("all")
```

Comment:
❓ Do we still need the `@SuppressWarnings("all")?

Reply:
I'll look into  this.